.packaging/conda_recipe/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
     - ninja
     - pkg-config  # [not win]
     - sysroot_linux-64 2.17  # [linux64]
+    - jsonschema
```

Comment:
This is not needed as a "build" dependency, which is for things that must run natively in a cross-compilation scenario and so includes mainly build tools. Having it in "host" is enough to be present for detection by CMake at build time, and "run" makes it a runtime dependency.

Reply:
cool! Will do. Question: what's the meaning of "build" then?